graphene_sqlalchemy/utils.py

```diff
@@ -25,8 +25,17 @@ def get_query(model, context):
 def is_mapped_class(cls):
     try:
         class_mapper(cls)
-    except (ArgumentError, UnmappedClassError):
-        return False
+    except (ArgumentError, UnmappedClassError) as error:
```

Comment:
I think it could be more pythonic written this way : 
```
    except ArgumentError as error:
        # Only handle ArgumentErrors for non-class objects
        if "Class object expected" in str(error):
            return False
        raise
    except UnmappedClassError:
        return False
```

Reply:
I appreciate your attention to detail, but is this nitpick worth keeping all of us users stuck on incorrect error messages for months?

Reply:
@heller It looks like there are merge conflicts so I think it's fine to address those nitpicks when fixing them. If you want to fix it and send a PR, I'm happy to merge it asap. Cheers.